states/menu.py

```diff
@@ -5,22 +5,29 @@
 
 
 class MenuOptions(StateOptions):
+    """Variables for Menu options"""
+
     BATTLE = 1
     TEAM = 2
     EXIT = 3
 
 
 class Menu(State):
+    """Displays options for Menu state change"""
+
     option: Optional[MenuOptions] = None
 
     def __init__(self):
+        """Logic for what occurs if exit option is not choesen"""
         super().__init__()
         while self.option is not MenuOptions.EXIT:
             print(TEXT["MAIN"]["ENTRY"])
             self.option = self.check_input(MenuOptions)
             self.choose_option()
 
     def choose_option(self):
+        """Runs respection option chosen"""
```

Comment:
I don't think respection is a word

Reply:
This is not a word. Changed to `respective`